client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/cloud-project-compute-infrastructure-virtualMachine-addEdit.html

```diff
@@ -1218,6 +1218,44 @@ <h4 data-translate="cpcivm_addedit_advanced_options_description"></h4>
                                                 </span>
                                             </td>
                                         </tr>
+                                        <!-- available but not activated regions -->
+                                        <tr data-ng-repeat="region in VmAddEditCtrl.availableRegions.data"
+                                            style="cursor: not-allowed;"
+                                            data-ng-if="!VmAddEditCtrl.availableRegions.loading && VmAddEditCtrl.availableRegions.data.length > 0">
+                                            <td style="width: 50px;">
+                                                <flat-radio>
+                                                    <input type="radio" aria-hidden="true" tabindex="-1"
+                                                           name="regionsChoice" id="{{::region.name}}"
+                                                           data-ovh-stop-event="click"
+                                                           disabled="true"
+                                                           value="{{::region.name}}">
+                                                </flat-radio>
+                                            </td>
+                                            <td class="space-p8 flag-box flag-box-cell" style="width: 50px;">
+                                                <div class="popover-list-item">
+                                                    <div class="popover-list-item-illustration">
+                                                        <i class="flag-icon {{:: VmAddEditCtrl.regionService.getRegionIconFlag(region.name) }} flag"></i>
+                                                    </div>
+                                                </div>
+                                            </td>
+                                            <td class="left-space-p4">
+                                                <span class="fs14 no-space normal col-xs-12" data-ng-bind="::VmAddEditCtrl.regionService.getTranslatedMicroRegion(region.name)"></span>
```

Comment:
If those class are really needed could they be replaced by bootstrap or UI Kit classes